js/ui/screenShield.js

```diff
@@ -187,6 +191,13 @@ var ScreenShield = class {
         });
     }
 
+    _clearCurrentDialog() {
```

Comment:
nit: a few lines above there's a block that could be replaced with `this._clearCurrentDialog()`

Reply:
Indeed. Done that.